app/helpers/application_helper.rb

```diff
@@ -0,0 +1,28 @@
+module ApplicationHelper
+  def flash_class(level)
+    case level
+    when :notice then
```

Comment:
You can avoid using case when with hashmap like e.g.: 
```ruby
classes = {
 notice: 'alert alert-info',
 success: 'alert alert-success',
....
}
```
And then simply use `classes[level]`.